### DISCUSSION TITLE
Xdrip reading Juggluco data

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#3788] [General]
I have xDrip reading data from Juggluco (using Libre).  The xDrip readings seem to be about 15 minutes delayed compared to Juggluco.  I would have expected the delay to be > 5 minutes.  Can anyone shed some light on this?

**Comment by PhilTU** [comment]
There is smoothing in xDrip for Libre 2 patched app input. For best response just use Juggluco alone.

Smoothing can be adjusted https://xdrip.readthedocs.io/en/latest/install/libre2patch/#smoothing

**Comment by GLoren1** [comment]
Thanks.  I would use only Juggluco, but I need the better alarm functions of xDrip.  Are there any plans for xDrip to support  Libre U.S. devices?

**Reply by zwidow** [reply]
you can try to enable plugin calibration, it should bypass the smoothing timeout. I didn't test if works only by enabling it and use xdrip original instead other plug-in. You can try
